api/v1alpha1/queue_types.go

```diff
@@ -32,6 +32,9 @@ type CapacityReference string
 
 // QueueStatus defines the observed state of Queue
 type QueueStatus struct {
+	// PendingWorkloads is the number of workloads currently assigned to this
+	// queue not yet assigned to a Capacity.
+	PendingWorkloads int32 `json:"pendingWorkloads"`
```

Comment:
Missing the started workloads and usage, has another merge PR?

Reply:
Will be another PR, I want this first one to lay down the status udpate to CR, with pending as the init example, a following PR will add more details to the kueue.queue.status field